encoding/video-artifacts.md

```diff
@@ -311,8 +311,8 @@ simply adjust the `min/max_in/out` parameters accordingly.
 
 [^3]: Blocking may also occur for other reasons other than compression data loss. [Image re-construction with padding][waifu2x238] can cause very similar looking effects, although this is irrelevant for fansubbing source videos.
 
-[^4]: The 8-bit limited range (used in rec.601 and rec.709) (and also BT.2020/2100) only defines values within \[16,235\] for the Y and \[16,240\] for the U and V planes. This means that Y=16 is considered full black and Y=235 full white, while any values outside of that range are clamped virtually (during rendering). U and V behave analogously. 
+[^4]: The 8-bit limited range (used in rec.601 and rec.709) (and also BT.2020/2100) only defines values within \[16,235\] for the Y and \[16,240\] for the U and V planes. This means that Y=16 is considered full black and Y=235 full white, while any values outside of that range are clamped virtually (during rendering). U and V behave analogously.
 
-[^5]: The limited ranges in different precisions are shifted by (multiplied by 2 to the power of) the added bits. For 12-bit, for example, you muliply by 2\^(12-8), resulting in \[256,3760\] and \[256-3840\] respectively. The maximum value in full range is obviously the highest unsigned integer value, so 2\*\*12-1.
+[^5]: The limited ranges in different precisions are shifted by (multiplied by 2 to the power of) the added bits. For 12-bit, for example, you multiply by $$2^{12-8}$$, resulting in \[256,3760\] and \[256-3840\] respectively. The maximum value in full range is obviously the highest unsigned integer value, so $$2^{12-1}$$.
```

Comment:
`$$2^{12}-1$$`

Reply:
![image](https://user-images.githubusercontent.com/10972040/66679287-de669080-ec3b-11e9-8a84-6c42b51114ae.png)
there's the three style options for the ranges, what we have now, sans-serif, and normal tex math font. I think I like what we have now better actually.

Reply:
I prefer the sans-serif version.